index.html

```diff
@@ -184,11 +184,13 @@ <h2 class="marketing-strategies__title">Marketing Strategies</h2>
       <footer class="footer">
         <div class="footer__wraper">
           <div class="footer__logo">
-            <img src="./assets/icons/Group 31.svg" alt="Logo">
-            <img src="./assets/icons/Productly.svg" alt="LogoText">
+            <a href="#">
+              <img src="./assets/icons/Group 31.svg" alt="Logo">
+              <img class="footer__logo-text" src="./assets/icons/Productly.svg" alt="LogoText">
```

Comment:
`img` is self-closing tag 
add `..... />`

Reply:
Completed

Reply:
completed